### ISSUE TITLE
ModuleNotFoundError: No module named 'streamlit.report_thread' on Streamlit 1.4.0

### ISSUE THREAD

**Issue by andfanilo** [whitphx/streamlit-webrtc#598]
Context: https://discuss.streamlit.io/t/modulenotfounderror-no-module-named-streamlit-report-thread/20983/2 

The following code:

```python
import streamlit as st 
from streamlit_webrtc import (
    AudioProcessorBase,
    RTCConfiguration,
    VideoProcessorBase,
    WebRtcMode,
    webrtc_streamer,
)
```

does not work on Streamlit 1.4.0

Streamlit 1.4.0 has been seen to break the `import streamlit.report_thread as ReportThread` import, I saw it twice in the codebase so I'm supposing this is the reason. Need to check with ST engineers if import is now private or renamed.

**Comment by whitphx** [comment]
Thank you for the report.
It seems to be because the module has been renamed in https://github.com/streamlit/streamlit/pull/4141 .
I will fix this extension to be compatible with it.

**Comment by whitphx** [comment]
will be fixed in 0.34.1
